flink-tests/src/test/java/org/apache/flink/test/state/operator/restore/keyed/KeyedJob.java

```diff
@@ -100,9 +100,7 @@ public static void main(String[] args) throws Exception {
 			.map(new StatefulStringStoringMap(mode, "first"))
 			.setParallelism(4);
 
-		if (mode == ExecutionMode.MIGRATE || mode == ExecutionMode.RESTORE) {
```

Comment:
@zentol can you confirm whether these changes make sense?

From the discussions I see in https://github.com/apache/flink/pull/3844, I assume this was a leftover to write a 1.2 savepoint (when uids couldn't be added for each chained operator separately). So it should be ok to remove this?

Reply:
yeah it should be alright to remove that, but let's chain the uid call to the operator creation as we do for the others for style points.